test/rentlinx/test_client.rb

```diff
@@ -3,6 +3,43 @@
 require_relative '../helper'
 
 # Test the Rentlinx Client
-class TestRentlinxClient < MiniTest::Test
+class ClientTest < MiniTest::Test
   include SetupMethods
+
+  def test_post
+    VCR.use_cassette('test_post') do
```

Comment:
I'm providing this comment in the context that the top level functions are as is (not with my former suggestions). Is it likely that there would be other `test_post` type methods? If so, the cassette name should probably be more specific so as to not conflict.